test/expect/tslint.json

```diff
@@ -1,8 +1,5 @@
 {
   "rulesDirectory": ["../../bin/rules"],
-  "linterOptions": {
```

Comment:
tslint doesn't need this any more, now it needs a `tsconfig.json` to be present. palantir/tslint#2769